webgpu.h

```diff
@@ -282,6 +282,7 @@ typedef enum WGPUTextureFormat {
     WGPUTextureFormat_BC6HRGBSfloat = 0x00000031,
     WGPUTextureFormat_BC7RGBAUnorm = 0x00000032,
     WGPUTextureFormat_BC7RGBAUnormSrgb = 0x00000033,
+    WGPUTextureFormat_None = 0xFFFFFFFF,
```

Comment:
@kainino0x what do you think of renaming this "Default" and set it at position 0 so that 0-initializing `WGPUTextureViewDescriptor` does what's expected?

Reply:
Great, let's do it! (sorry I didn't catch that earlier on the Dawn review)